src/DetailsView/components/tab-stops/tab-stops-failed-instance-panel.tsx

```diff
@@ -47,8 +47,8 @@ export const TabStopsFailedInstancePanel = NamedFC<TabStopsFailedInstancePanelPr
             confirmButtonText: 'Add failed instance',
             onConfirm: () => {
                 tabStopRequirementActionMessageCreator.addTabStopInstance({
-                    description: failureInstanceState.description,
-                    requirementId: failureInstanceState.selectedRequirementId,
+                    description: failureInstanceState.description!,
```

Comment:
This is only called when the form is submitted, and the form can only be submitted once values for these attributes are provided.